### PR TITLE
Fixing TPU scripts

### PR DESCRIPTION
**Pull request by vjeronymo2**

### CODE REVIEW

File: tpu/create_msmarco_monot5_input.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+
+def load_corpus(path):
+    print("Loading corpus...")
+    corpus = {}
+    if ".json" in path:
+        with jsonlines.open(path) as reader:
+            for obj in tqdm(reader):
+                id = obj["id"]
+                # corpus[id] = obj['contents']
```
Comment (rodrigonogueira4) [inline tpu/create_msmarco_monot5_input.py:47]: remove commented code?

File: tpu/create_msmarco_monot5_input.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
+"""
```
Comment (rodrigonogueira4) [inline tpu/create_msmarco_monot5_input.py:1]: Maybe rename the file to "create_msmarco_monot5_inference.py as there is another one called "create_msmarco_monot5_train.py"

File: tpu/get_t5_3B_inpars.py [hunk 3]
```diff
@@ -11,62 +11,52 @@
 parser.add_argument(
     "--variation",
     type=str,
-    default="",
+    default="inpars_10k",
```
Comment (rodrigonogueira4) [inline tpu/get_t5_3B_inpars.py:14]: maybe rename to "exp_name"?

File: tpu/run_t5_3B_inpars.py [hunk 4]
```diff
@@ -11,7 +11,7 @@
 parser.add_argument(
     "--variation",
```
Comment (rodrigonogueira4) [inline tpu/run_t5_3B_inpars.py:12]: same here

**Comment by rodrigonogueira4** [comment]
Thanks a lot for doing this!
Could you please also add the missing ftfy package in the requirements?
And remove the duplicated pyserini in the requirements? (cc @lhbonifacio @hugoabonizio)

File: tpu/create_msmarco_monot5_input.py (hunk 2 above)
Comment (vjeronymo2) [reply tpu/create_msmarco_monot5_input.py:1]: I can do that
just FYI, "create_msmarco_monot5_input.py" is the same name as in pyserini's repo

**Comment by vjeronymo2** [comment]
All done

File: tpu/get_t5_3B_inpars.py [hunk 5]
```diff
@@ -11,62 +11,52 @@
 parser.add_argument(
     "--variation",
     type=str,
-    default="",
+    default="exp_name",
```
Comment (rodrigonogueira4) [inline tpu/get_t5_3B_inpars.py:14]: sorry, I meant the argument name as "exp_name"

**Review by rodrigonogueira4 — APPROVED** [review]
Thanks for doing this! I just left one minor comment ...

**Pull request closed without merging**